coursebook/modules/m2/2-hands-on.ipynb

```diff
@@ -8,12 +8,14 @@
     "\n",
```

Comment:
<p>Great point with the values representing NA</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/alan-turing-institute/rds-course/pull/86/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>